common/djangoapps/student/views.py

```diff
@@ -1789,7 +1790,7 @@ def create_account_with_params(request, params):
         subject = ''.join(subject.splitlines())
         message = render_to_string('emails/activation_email.txt', context)
 
-        from_address = microsite.get_value(
+        from_address = theming_helpers.get_value(
```

Comment:
Here we are referencing "email_from_address" and not "default_from_email" -- seems like we should be consistent with the field we support instead of having two fields which can be confused.  Perhaps we can check with @arbabnazar to see what the values look like in our production microsite configurations.